backend/app/controllers/api/v1/accounts/open_call_applications_controller.rb

```diff
@@ -5,6 +5,20 @@ class OpenCallApplicationsController < BaseController
         around_action(only: %i[create]) { |_, action| set_locale(language: current_user&.account&.language, &action) }
         load_and_authorize_resource
 
+        def index
+          @open_call_applications = @open_call_applications.includes(open_call: {investor: :account}, project: {project_developer: :account})
+          @open_call_applications = @open_call_applications.where project_id: filter_params[:project_id] if filter_params[:project_id].present?
```

Comment:
I did not find a requirement for this and to filter by open_call_id but I trust this is needed :)

Reply:
Ahh, I guess this is for details endpoint, to get only those for project/open call.

Reply:
or should we add a separate show endpoint for details?